protocol/src/main/java/net/md_5/bungee/protocol/packet/Login.java

```diff
@@ -135,7 +136,13 @@ public void read(ByteBuf buf, ProtocolConstants.Direction direction, int protoco
             dimension = readString( buf );
             worldName = readString( buf );
             seed = buf.readLong();
-            gameMode = buf.readUnsignedByte();
+            if ( protocolVersion >= ProtocolConstants.MINECRAFT_1_20_5 )
```

Comment:
Can probably just leave this, don't think it's actually changed and will never matter